pkg/dev/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM linuxkit/sshd:e108d208adf692c8a0954f602743e0eec445364e
```

Comment:
Its better to duplicate the `sshd` definition rather than inheriting, as we will probably want other changes, while `sshd` is just a minimal ssh (and we might remove it)